### DISCUSSION TITLE
VimTex Integration

### DISCUSSION THREAD

**Discussion by Deeknskeet10** [NvChad/NvChad#640] [Q&A] [answered]
Hey all,

Attempting to add VimTex to the current NvChad configuration and was looking for some guidance.  I believe I can just add `use {"lervag/vimtex"}` to the pluginList.lua and then update my chadrc.lua with the vimtex configuration requirements.  Are their further modifications required?  Desire is to create a workflow for writing LaTex docs alongside a viewer like Zathura.  Does NvChad have a native solution that I'm missing?

Thanks!

**Comment by siduck** [comment, marked as answer]
not pluginlist.lua! you are not supposed to edit files outside custom dir 

check https://nvchad.github.io/config/walkthrough

**Reply by Deeknskeet10** [reply]
Glad I asked first ha, thank you!
